client-next/components/dashboard/link_tree.tsx

```diff
@@ -1,4 +1,6 @@
+import { useEffect, useState } from "react";
 import { Link } from "../../near/types";
+import ModalEditLink from "../modal/modal_edit_link";
 import LinkTreeItem from "./link_tree_item";
```

Comment:
Rename this file to `link3.tsx`.

We do not want any name related to "link_tree" in the project.